TfsMigrate.Processors.WorkItemTracking/WorkItemTrackingProcessor.cs

```diff
@@ -24,16 +22,17 @@ namespace TfsMigrate.Processors.WorkItemTracking
 {
     public class WorkItemTrackingProcessor : Processor
     {
-        protected override async Task InitializeCoreAsync ( CancellationToken cancellationToken )
+        protected override async Task InitializeCoreAsync(CancellationToken cancellationToken)

```

Comment:
Please do not change the styling of this file. The styling should follow what is already there.